### ISSUE TITLE
[1337x] (testing) Exception (1337x): certificate validation failed: [Subject]  E=root@localhost.localdomain, CN=localhost.localdomain, OU=IT, O=MyCompany, L=Seattle, S=WA, C=US[Issuer]  E=root@localhost.localdomain, CN=localhost.localdomain, OU=IT, O=MyCompany, L=Seattle, S=WA, C=US[Serial Number]  103511C7[Not Before]  8/14/2018 1:30:15 PM[Not After]  8/11/2028 1:30:15 PM[Thumbprint]  336EAC25B38A5B1878C9615F5AFF6D973B4CA1C0: The SSL connection could not be established, see inner exception.

### ISSUE THREAD

**Issue by miniliQuid** [Jackett/Jackett#13216]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: No
Using proxy: Disabled
App config/log directory: C:\ProgramData\Jackett
ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads
Running in Docker: No
Jackett variant: CoreWindows
OS version: Microsoft Windows NT 10.0.19043.0 (64bit OS)
Environment version: 6.0.4 (C:\ProgramData\Jackett\)
Jackett v0.20.986
```

### Description

1337x gives an error exception about the certificate. In the troubleshooting area there is a tip on how to retry requesting a certificate, but besides that I am not super handy with these things, the main problem is that there is no windows option mentioned.

Now this isn't the only site not working, but I hope to be able to fix most of those once I get how to solve 1 issue.
Majority of the added sites on jackett work perfectly fine by the way, just a few exceptions of which this is for me the most important one.

### Logged Error Messages

```text
Exception (1337x): certificate validation failed: [Subject] E=root@localhost.localdomain, CN=localhost.localdomain, OU=IT, O=MyCompany, L=Seattle, S=WA, C=US [Issuer] E=root@localhost.localdomain, CN=localhost.localdomain, OU=IT, O=MyCompany, L=Seattle, S=WA, C=US [Serial Number] 103511C7 [Not Before] 8/14/2018 1:30:15 PM [Not After] 8/11/2028 1:30:15 PM [Thumbprint] 336EAC25B38A5B1878C9615F5AFF6D973B4CA1C0 : The SSL connection could not be established, see inner exception.
```

**Comment by ilike2burnthing** [comment]
Are you able to access the same URL in browser on the same Windows device and IP as Jackett? If not, try the alternate URLs for one which works for you in browser, or use a VPN or proxy.

**Comment by garfield69** [comment]
@miniliQuid The certificate you are seeing is most likely the result of your ISP blocking your access to 1337x
Or possibly from a security app (anti-malware, firewall, etc) on your PC or router.
IF you are also getting these errors for other indexers then take a look at the thumbprint of the certificate generating the error, it they are all 336EAC25B38A5B1878C9615F5AFF6D973B4CA1C0 then that confirms that something is blocking your access, since the CA for all sites should be unique to each site.
As ilike2burnthing stated earlier, changing your DNS from the one provided by your ISP to one of the public ones such as Cloudflare's 1.1.1.1 or Google's 8.8.8.8 is usually effective for ISP blocking circumvention, and using a proxy or VPN is typically the most effective option.

**Comment by miniliQuid** [comment]
Thanks for the swift replies both of you.
1337x has been blocked in my country by many providers (I guess some others may be blocked as well).
However I am able to visit the site as my browser goes through the VPN.

I think I have added jackett to the VPN as well, so shouldn't that help jackett to be able to access 1337x as well?
I will check it to make sure I did it all correctly, cause perhaps I have made a mistake, or perhaps it is simply not that easy?

**Comment by ilike2burnthing** [comment]
I take it you're using some sort of split tunnelling with your VPN for Jackett and your browser?

Try disabling your VPN, just change the DNS for your network adapter, restart Jackett, see if it works. Failing that, disable split tunnelling for your VPN, enable your VPN, restart Jackett, see if that works.

**Comment by miniliQuid** [comment]
> I take it you're using some sort of split tunnelling with your VPN for Jackett and your browser?
> 
> Try disabling your VPN, just change the DNS for your network adapter, restart Jackett, see if it works. Failing that, disable split tunnelling for your VPN, enable your VPN, restart Jackett, see if that works.

ah yes I do, and a few other programs, but there are some programs not working properly when behind VPN, so yeah that is why I made it do a split and simply have the programs that need it tunnel through the VPN.

I don't think I can change the DNS since I need to be able to connect for work (and some other things) when I am not at home. Doesn't changing DNS mess that up? AFAIK I needed a static IP and ports forwarded to be able to do all the things I need to do.

Your last suggestion definitely worked. The only problem is that it then creates a problem for the other programs as mentioned earlier :(

Any clue why jackett ignores the splittunnel, but works fine when VPN is just on and not using split tunnel function?

**Comment by ilike2burnthing** [comment]
If you don't use the VPN for Jackett, do you get exactly same error?

What Jackett executable have you put behind the VPN? You may have selected the wrong one, try the others.

Unless you're using a custom DNS to be able to connect to your work servers, I don't see why using 1.1.1.1 or 8.8.8.8 would affect it.

**Comment by miniliQuid** [comment]
Thanks for the swift replies, you guys are a great help.
In case this may help others in the future, here are some answers and the last things I tried, and I think your middle sentence figured out the problem.

With jackett as one of the programs using splittunnel it didnt work (probably like you mentioned I used wrong executable).
While the VPN was on completely, jackett worked and gave no error (other programs had problems then though).
However not that instead of adding jackett, and download programs alike, as a line to use the VPN, I reversed it and made everything use VPN unless the program was added to the exceptions list of the VPN.

So now I simply added the programs that shouldn't use the VPN, instead of the programs that should.

So far everything seems to work as intended now 😃 

Thanks you guys!
